src/test/regress/expected/multi_mx_create_type.out

```diff
@@ -0,0 +1,35 @@
+-- Tests related to create type
+CREATE TYPE create_order_delegator_t AS (
+     out_w_tax decimal(4, 4),
+     out_d_tax decimal(4, 4),
+     out_o_id integer,
+     out_c_discount decimal(4, 4),
+     out_c_last varchar(16),
+     out_c_credit char(2),
+     out_o_entry_d timestamp
+);
+\d create_order_delegator_t
+               Composite type "public.create_order_delegator_t"
+     Column     |            Type             | Collation | Nullable | Default
+---------------------------------------------------------------------
+ out_w_tax      | numeric(4,4)                |           |          |
+ out_d_tax      | numeric(4,4)                |           |          |
+ out_o_id       | integer                     |           |          |
+ out_c_discount | numeric(4,4)                |           |          |
+ out_c_last     | character varying(16)       |           |          |
+ out_c_credit   | character(2)                |           |          |
+ out_o_entry_d  | timestamp without time zone |           |          |
```

Comment:
\d output is prone to changes across postgres versions. Better to use something like `SELECT "Column", "Type", "Modifiers" FROM table_desc`

Reply:
I've gone ahead & removed this file altogether